website/docs/en/contribute/development/releasing.md

```diff
@@ -46,8 +46,8 @@ Take pnpm as an example:
 {
   "pnpm": {
     "overrides": {
-      "@rspack/binding": "npm:@rspack/binding-canary@nightly",
-      "@rspack/core": "npm:@rspack/core-canary@nightly",
+      "@rspack/binding": "npm:@rspack-canary/binding@nightly",
```

Comment:
Overriding `@rspack/binding` seems unnecessary, it is a sub-dependency of `@rspack/core`